### DISCUSSION TITLE
〔 Comments Section 〕Fallout Shelter Online: Introduction

### DISCUSSION THREAD

**Discussion by morgosus** [morgosus/morgosus.github.io#10] [Comments]
A comments section for the [Fallout Shelter Online: Introduction](https://www.toms.click/gaming/foso/2021/08/13/fallout-shelter-introduction.html) post.

**Comment by Jalquin** [comment]
@morgosus told me bout this game today morning. If I had more time to spend on gaming I will surely try this one!

**Comment by Amadikasun** [comment]
It's a great game that was some time ago recommended to me by Morgosus, but I didn't have time until now. I definitely should have started playing sooner!
